packages/vats/src/core/basic-behaviors.js

```diff
@@ -489,7 +489,8 @@ export const makeAddressNameHubs = async ({
 
   const perAddress = address => {
     const { nameHub, myAddressNameAdmin } = makeMyAddressNameAdminKit(address);
-    myAddressNameAdmin.reserve(WalletName.depositFacet);
+    // This may not finish before another reserve() call, but this one will win
+    void myAddressNameAdmin.reserve(WalletName.depositFacet);
```

Comment:
`makeMyAddressNameAdminKit` is deprecated, yes? I'm not confident how this solo client stuff should work. I'd be inclined to have ci continue to gripe about it. Or leave an `XXX` or something.